frontend/pages/_document.js

```diff
@@ -22,6 +22,10 @@ class MyDocument extends Document {
             href="https://fonts.googleapis.com/icon?family=Material+Icons|Lato:400,700&display=swap"
             rel="stylesheet"
           />
+          <script
+            src="https://kit.fontawesome.com/4cbc89a501.js"
+            crossorigin="anonymous"
+          ></script>
```

Comment:
You could use the fontawesome npm package rather than this way. The reason you want to do that is because webpack can do some fancy stuff like tree shaking and code splitting to reduce bundle size. If you import it like this you cant and it will be downloaded for all pages even pages that don't need fontawesome

Reply:
Interesting... now using fontawesome npm package. Plz review.